airbyte-integrations/connectors/source-pivotal-tracker/source_pivotal_tracker/source.py

```diff
@@ -123,6 +134,16 @@ def _get_authenticator(config: Mapping[str, Any]) -> HttpAuthenticator:
         token = config.get("api_token")
         return PivotalAuthenticator(token)
 
+    @staticmethod
+    def _check_project_availability(auth: HttpAuthenticator, project_id) -> bool:
+        try:
+            project = Project(authenticator=auth, project_id=project_id)
+            next(project.read_records(SyncMode.full_refresh))
+        except Exception as err:
+            logger.error("Unable to fetch project info: %s", err)
```

Comment:
Are we able to explicitly mention that this project will be skipped?

Reply:
An error will make the sync fails, it is this expected?

Reply:
Also make sure the `Project` class has the cache enabled. If not you'll make an API request to every record.